lib/src/__tests__/setup.ts

```diff
@@ -6,18 +6,6 @@ const EnzymeAdapter = require('enzyme-adapter-react-16');
 // Setup enzyme's react adapter
 Enzyme.configure({ adapter: new EnzymeAdapter() });
 
-// Fix popper mount https://github.com/mui-org/material-ui/issues/15726
-// TODO, upgrade jsdom to 16.0.0 and remove this line.
```

Comment:
cc @dmtrKovalenko, it's gone, one step more aligned with the main repository :).